### DISCUSSION TITLE
How do add history when creating session/context

### DISCUSSION THREAD

**Discussion by carlosgalveias** [withcatai/node-llama-cpp#107] [Q&A] [answered]
I am doing a chatbot and on the frontend side, I'm saving the conversations I am having. Now I want to resume a previous conversation and I would like to send to my backend the history .
I am cleaning the previous session and context and making a new one, but how do I create it with already a history? 

PS: I am using the beta

**Comment by giladgd** [comment, marked as answer]
You can use the [`conversationhistory` parameter](https://withcatai.github.io/node-llama-cpp/api/type-aliases/LlamaChatSessionOptions#conversationhistory) when creating a new `LlamaChatSession` object
